server/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -969,6 +969,21 @@ public void testRepeatedFetchDoesNotLeakSysJobsLog() throws Exception {
         }
     }
 
+    @Test
+    public void testExecuteBatchDoesNotLeakSysJobsLog() throws Exception {
+        try (Connection conn = DriverManager.getConnection(url(RW), properties)) {
+            PreparedStatement stmt = conn.prepareStatement("SELECT ?");
+            for (int i = 1; i <= 3; i++) {
+                stmt.setInt(1, i);
+                stmt.addBatch();
+            }
+            stmt.executeBatch();
+        }
+        for (JobsLogService jobsLogService : cluster().getDataNodeInstances(JobsLogService.class)) {
+            assertBusy(() -> assertThat(jobsLogService.get().activeJobs()).isEmpty());
+        }
+    }
+
```

Comment:
In the flink repro I didn't use batching options, ie smth like

```
JdbcExecutionOptions.builder()
                                    .withBatchSize(10)
                                    .withBatchIntervalMs(1000)
```

but those are provided by default (https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/jdbc/#jdbc-execution-options), so finally managed to get a reproduction with tests